R/read_vpts.R

```diff
@@ -1,241 +1,144 @@
-#' Read a time series of vertical profiles (`vpts`) from file
-#'
-#' @param file A text file containing the standard output (stdout) generated
-#' by vol2bird (or the package function `calculate_vp`).
-#' @param lon numeric. Longitude of the radar in decimal degrees.
-#' @param lat numeric. Latitude of the radar in decimal degrees.
-#' @param height numeric. Height above sea level of the radar antenna in meters.
-#' @param radar A string containing a radar identifier.
-#' @param wavelength Radar wavelength in cm, or one of 'C' or 'S' for C-band
-#' and S-band radar, respectively, in which case C-band wavelength is assumed
-#' to be 5.3 cm and S-band wavelength 10.6 cm
-#' @param sep the field separator character, see [utils::read.table]
-#'
-#' @return An object inheriting from class `vpts`, see
-#' [`vpts()`][summary.vpts] for details.
+#' Read time series of vertical profiles (`vpts`) from file(s)
 #'
+#' Reads `vpts` data from one or more files.
+#' The following file formats are supported (but cannot be mixed):
+#' - [VPTS CSV](https://aloftdata.eu/vpts-csv/).
+#' - [ODIM bird profile](https://github.com/adokter/vol2bird/wiki/ODIM-bird-profile-format-specification).
+#' @param files Path(s) to one or more files containing vpts data.
+#' @param ... Additional arguments for backward compatibility, passed to `read_stdout`.
+#' @return `vpts` object.
+#' @family read functions
 #' @export
-#'
-#' @examples
```

Comment:
each function needs examples demonstrating most of the functionality

Reply:
@peterdesmet @iskandari Suggest adding a short VPTS CSV file in `/inst/extdata` that we can use in examples. Helpful if you pick an example file that also has some interesting bird profile in it, so it can be used in other examples as well. Can be a single night with a little lead and end time to keep the package small.

Reply:
Added example prototypes in 6d2311d7edb02061f09c934ca9dbdb7e81df1874, these have several FIXMEs that still need to be addressed

Reply:
Just reading that in tests you download VPTS CSV files. You could do the same in the example (instead of including file in `/inst/extdata`, but then that part of the example should be in a \dontrun{} block, because we can't assume internet connectivity

Reply:
Indeed, you can read remotely, so no need to store locally. You can look on https://crow.aloftdata.eu/#/ for a profile of interest